server/routes/caseNotes/caseNotesController.ts

```diff
@@ -14,7 +22,26 @@ export default class CaseNotesController {
       size: 5,
     }
     const caseNotesPage = await this.interventionsService.getCaseNotes(accessToken, req.params.id, paginationQuery)
-    const presenter = new CaseNotesPresenter(caseNotesPage)
+    const uniqueUsers = caseNotesPage.content
```

Comment:
a Set is a more natural collection for this - it removes the requirement to filter dups too

Reply:
Changed to a set (and there is a test to make sure only one call is made)